pkgs/development/compilers/llvm/15/clang/default.nix

```diff
@@ -36,9 +36,14 @@ let
       "-DSPHINX_OUTPUT_MAN=ON"
       "-DSPHINX_OUTPUT_HTML=OFF"
       "-DSPHINX_WARNINGS_AS_ERRORS=OFF"
-    ] ++ lib.optionals (stdenv.hostPlatform != stdenv.buildPlatform) [
+    ] ++ lib.optionals (!stdenv.buildPlatform.canExecute stdenv.hostPlatform) [
```

Comment:
Broke `pkgsCross.musl64.llvmPackages_16.clang.cc`  and `pkgsStatic.llvmPackages_16.clang.cc` (15 too)

Reply:
https://github.com/NixOS/nixpkgs/pull/250612